### ISSUE TITLE
Errors and warnings after install in Kali

### ISSUE THREAD

**Issue by Cix-16** [MISP/MISP#5779]
I got some errors and warnings when I installed in kali.:
1. I ran the command in root, but I got this:
```
Next step: Setting install options with given parameters.
id: ‘misp’: no such user
/tmp/misp-kali.sh: line 661: useradd: command not found
/tmp/misp-kali.sh: line 662: chpasswd: command not found
```
2. The Kali is in vmware environment ,but I got this:
```
Next step: We guess that this is a physical machine and cannot possibly guess what the MISP_BASEURL might be.
You can now enter your own MISP_BASEURL, if you wish to NOT do that, the MISP_BASEURL will be empty, which will work, but ideally you configure it afterwards.
```
3.There also seems to hava php version issues. The Kali version is 2020.1 and php version is 7.4. But redis and gnupg were install in php7.4, so I got follow problem:
```
/tmp/misp-kali.sh: line 2756: phpenmod: command not found
/tmp/misp-kali.sh: line 2757: phpenmod: command not found
```

**Comment by demiscuzz** [comment]
Did you find the solution? I am having more errors when running the script in kali. when the install finnishes I get a 501.

**Comment by SteveClement** [comment]
Fixed in #6952
